frontend/src/pages/IstioConfigNew/IstioConfigNewPage.tsx

```diff
@@ -143,6 +157,7 @@ export const NEW_ISTIO_RESOURCE = [
   { value: AUTHORIZACION_POLICY, label: AUTHORIZACION_POLICY, disabled: false },
   { value: GATEWAY, label: GATEWAY, disabled: false },
   { value: K8SGATEWAY, label: K8SGATEWAY, disabled: false },
+  { value: K8SREFERENCEGRANT, label: K8SREFERENCEGRANT, disabled: false },
```

Comment:
I think it is better to write variable name in snake case: K8S_REFERENCE_GRANT. The same should be applied to K8S_GATEWAY, but it is not in the scope of this PR.